onnxscript/rewriter/ort_fusions/skip_normalization.py

```diff
@@ -38,7 +38,7 @@ def _skip_rms_normalization(op, input, skip, gamma, epsilon, stash_type):
 
 
 def _skip_layer_norm_pattern(op, input, skip, gamma, beta, epsilon, stash_type):
-    skip_sum = op.Add(input, skip)
+    skip_sum = op.Add(skip, input)
```

Comment:
I thought Skip Normalization related fusions were moved into a separate PR? Are these fixes to that PR?

Reply:
Have reverted the change you highlighted above. However, the fusion pattern for the skip normalization fusion with bias was modified in this PR as the pattern previously was too general and was causing replacements to happen if when they shouldn't have.